Misc/NEWS.d/next/Library/2017-10-11-22-18-04.bpo-31764.EMyIkK.rst

```diff
@@ -0,0 +1,2 @@
+Prevent a crash in `sqlite3.Cursor.close()` in case the `Cursor` object is
```

Comment:
Please use double backticks (\`\`). I tried to explain why single backticks shouldn't be used at https://github.com/python/cpython/pull/3925#discussion_r143451549

Reply:
I am not familiar with Sphinx, but i trust you are :)
So maybe https://devguide.python.org/committing/#what-s-new-and-news-entries should also recommend double backticks?

Reply:
Yeah, I wasn't aware of that example in the devguide. I will update it, thanks!